sleap/gui/web.py

```diff
@@ -146,6 +149,49 @@ def get_release(self, version: str) -> Release:
         )
 
 
+@attr.s(auto_attribs=True)
+class AnnouncementChecker:
+    """Checker for new announcements on the bulletin page of sleap."""
+
+    bulletin_json_path: str = BULLETIN_JSON
+    previous_announcement_date: str = prefs.prefs["announcement last seen date"]
+
+    def check_for_announcement(self) -> bool:
+        """Returns if new announcements are available."""
```

Comment:
Technically always intended to return since we catch the error.
```suggestion
        """Returns True if new announcements are available, otherwise returns False."""
```